gluster-exporter/main.go

```diff
@@ -80,6 +81,13 @@ func main() {
 			}
 		}
 	}
+
+	// If GD2_ENDPOINTS env variable is set, use that info
```

Comment:
So ENV variable gets the highest priority?

Reply:
with the second approach, env variable will not be useful if the value in config is not empty(Which is default case)

Reply:
With the second method when do you set the empty string in config? Is it empty by default? The first method looks good to me though. The GD2_ENDPOINTS env variable having higher precedence than config in the GCS setup won't be an issue right?

Reply:
@aravindavk I feel its better to go ahead with approach one. I just wanted to understand so asked the question earlier.

Reply:
> With the second method when do you set the empty string in config? Is it empty by default? 

empty string can be set as we are altering other config options here https://github.com/gluster/glusterd2/pull/1308

> The first method looks good to me though. The GD2_ENDPOINTS env variable having higher precedence than config in the GCS setup won't be an issue right?

No issues.

Reply:
>  I feel its better to go ahead with approach one. I just wanted to understand so asked the question earlier.

cool